frontend/src/container/LogsExplorerViews/index.tsx

```diff
@@ -177,6 +189,14 @@ function LogsExplorerViews(): JSX.Element {
 		setActiveLog(null);
 	}, []);
 
+	const handleSetActiveContextLog = useCallback((log: ILog) => {
```

Comment:
can we move this logic to hook itself ?

Reply:
prop drilling would be avoided

Reply:
I'm not sure because the activeContextLog is saved locally as activeLog

Reply:
would putting `activeLog` from hooks works ?